run_scripts/run_tpcc_on_clients.cpp

```diff
@@ -0,0 +1,219 @@
+#include <iostream>
+#include <fstream>
+#include <vector>
+#include <numeric>
+#include <sstream>
+#include <stdio.h>
+#include <string.h>
+#include <thread>
+#include <chrono>
+
+const int total_warehouses = 100100; // 100000 for 12xl
+
+const int num_ips_per_client = 3; // 3 for 12xl
+const int client_repeat_count = 2; // 2 for 12xl
+
+const int loader_threads_per_client = 21; // 60 for 12xl
+const int num_connections_per_client = 200; // 300 for 12xl
+
+const int delay_per_client = 30;
+const int load_delay_per_client = 20;
+const bool ignore_masters = true;
```

Comment:
let's add more detailed comments for each of these.

Reply:
Done.